### DISCUSSION TITLE
Find the DH parameters for iCub Simulator Arm

### DISCUSSION THREAD

**Discussion by LakshithaHKUST** [robotology/community#101] [Q&A] [answered]
Hello Everyone,

I would like to know whether there is a way to find the DH parameters for the iCub simulator arm?
There seems to be different versions of the arm and I am little bit confused.

Thanks in advance.

**Comment by pattacini** [comment]
As per the [`code`](https://github.com/robotology/icub-main/blob/master/src%2Fsimulators%2FiCubSimulation%2Fodesdl%2FiCub.cpp#L3982), the kinematic version used by the simulator is the default one, that is `1.0`.

**Comment by pattacini** [comment, marked as answer]
To retrieve the corresponding DH parameters, you can refer to our [**wiki**](http://wiki.icub.org/wiki/ICubFowardKinematics_left).

**Comment by LakshithaHKUST** [comment]
Thank You very much for the prompt reply. I'll proceed with v1.0 DH parameters.

**Comment by pattacini** [comment]
@LakshithaHKUST 

You should have received an invitation to join our developers' group.
Once accepted you'll be entitled to assign **labels** to issues herein.

**Comment by LakshithaHKUST** [comment]
could you please send me the invitation again, my email had a typo
sorry about the inconvenience. thanks for adding me to the developers' group.

**Comment by pattacini** [comment]
Sure; just done!
